### PR TITLE
docs(component): updated readME and added test job folder

### PR DESCRIPTION
**Pull request by rangfeli**
Issue #, if available:

Description of changes:

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: solutions/swb-reference/src/HPC/README.md [hunk 1]
```diff
@@ -30,20 +30,19 @@ Scheduling:
...
           - subnet-abc123
-Region: us-west-2
+Region: us-east-1
 Image:
   Os: alinux2
 ```
-4. Create atleast one S3 bucket on your AWS Main Account containing files necessary to complete a job such as scripts and data. Add the `S3 Full Access` policy to the IAM role assumed by the head node of your cluster to have permission to execute S3 actions. Lastly, similar to the policy below, attach the appropriate bucket policy to your S3 bucket to allow your cluster head node to access the bucket.
-
-```
+5. Create atleast one S3 bucket on your AWS Main Account containing files necessary to complete a job such as scripts and data. Attach the `S3 Full Access` policy to both the IAM role of the head node (`clusterName-RoleHeadNode`) and the IAM role of the compute nodes (`clusterName-Role`) of your ParallelCluster for the nodes to execute S3 operations. Lastly, similar to the policy below, attach the appropriate bucket policy to your S3 bucket to allow your ParallelCluster to access the bucket.
```
Comment (nguyen102) [inline solutions/swb-reference/src/HPC/README.md:37]: Let's format it like this `<clusterName>-RoleHeadNode` and mention it's a prefix. Also mention that these roles are in the hosting account

File: solutions/swb-reference/src/HPC/README.md [hunk 2]
```diff
@@ -30,20 +30,19 @@ Scheduling:
...
+```json
 {
     "Version": "2012-10-17",
     "Statement": [
         {
             "Effect": "Allow",
             "Principal": {
-                "AWS": "arn:aws:iam::AWS-Hosting-Account-ID:Hosting-Cluster-InstanceProfileHeadNode-123456789"
+                "AWS": "arn:aws:iam::AWS-Hosting-Account-ID:role/parallelcluster/*"
```
Comment (nguyen102) [inline solutions/swb-reference/src/HPC/README.md:45]: Is only `/*` needed or `/*/*`? Just to confirm.

File: solutions/swb-reference/src/HPC/artifacts/test_job/test.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+######################
+#---Example--Code----#
+srun hostname #Prints to StandardOutput name of compute node working on job
+cp ../data.txt data_copy.txt #Creates copy of example data.txt and places it in output
+#---Example--Code----#
+######################
+#Everything in s3_folder_name/output is copied back to same S3 bucket
+#Ensure bucket name is correct for your account below
+aws s3 cp . s3://main-account-bucket/test_job/output/ --recursive
```
Comment (nguyen102) [inline solutions/swb-reference/src/HPC/artifacts/test_job/test.sh:10]: Nitpick
`s3://main-account-bucket/test_job/output/` => `s3://<bucketName>/test_job/output/`

File: solutions/swb-reference/src/HPC/README.md (hunk 2 above)
Comment (rangfeli) [reply solutions/swb-reference/src/HPC/README.md:45]: /*/* is correct

**Review by nguyen102 — APPROVED** [review]

**Pull request merged**